xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/LivedataDropdownMenu.vue

```diff
@@ -103,6 +103,13 @@
         </a>
       </li>
 
+      <li v-for="extensionPanel in logic.extensionPanels" :key="extensionPanel.id">
```

Comment:
I would have preferred to have ``logic.panels``, initialized by default with the properties, sort and filter panels. Then others could simply call ``logic.addPanel`` to add additional panels.

Reply:
As I've mentioned in the comment above, I also think this could be a good idea. Not doing this now felt a bit safer in terms of not introducing any race conditions or other regressions. The question for me is still what we would store for them, there would definitely not be any HTML element but just a type, the question is if they should still get a name, title and icon? And if yes, how do we get the translated values in there?